packit_service/worker/checker/testing_farm.py

```diff
@@ -148,3 +148,25 @@ def pre_check(self) -> bool:
             f"!= comment:${self.testing_farm_job_helper.comment_arguments.identifier})"
         )
         return False
+
+
+class IsLabelFromCommentMatching(Checker, GetTestingFarmJobHelperMixin):
+    """
+    Check that job label is matching comment --labels option when it is specified.
+    If labels are not specified it will allow all jobs execution,
+    otherwise only jobs with the same label.
+    """
+
+    def pre_check(self) -> bool:
+        if not self.testing_farm_job_helper.comment_arguments.labels or any(
+            x in self.testing_farm_job_helper.comment_arguments.labels
+            for x in self.job_config.labels
+        ):
```

Comment:
I am not sure if we should go with AND or OR when multiple labels are specified. Have you thought about that? Do you have any reasoning behind that? (I am really on the fence here.)